jsonschema-core.xml

```diff
@@ -1333,26 +1335,31 @@
                     <t>
                         If present, the value for this keyword MUST be a string, and MUST represent a
                         valid <xref target="RFC3986">URI-reference</xref>.  This URI-reference
-                        SHOULD be normalized, and MUST resolve to an
-                        <xref target="RFC3986">absolute-URI</xref> (without a fragment).  Therefore,
-                        "$id" MUST NOT contain a non-empty fragment, and SHOULD NOT contain an
-                        empty fragment.
+                        SHOULD be normalized, and MUST be semantically equivalent to an
+                        <xref target="RFC3986">absolute-URI</xref> (without a fragment).
```

Comment:
@jdesrosiers it might be better to keep the original wording, yeah.  The reason I changed it is that media type-specific fragment transformations aren't really a part of URI "resolution."  But as @karenetheridge notes, "semantically equivalent" might be a little obscure.  It just means that the underlying meaning of the URI is the same.  Although really even that might be debatable as without a fragment identifies the resource, and with a fragment identifies the root object, which happens to effectively be the same thing.

Meh.  Yeah, maybe "resolve to" is more clear and as close to correct as anything?

Reply:
I tried to sort this out better, not sure if I succeeded or not.

Reply:
You definitely succeeded. The new text says what it needs to say and is easy to read.